.github/workflows/manual.yml

```diff
@@ -1,23 +1,67 @@
-# This is a basic workflow that is manually triggered
 
-name: Manual workflow
+name: TABTestKit CI
 
 on:
   workflow_dispatch:
+  pull_request:
+    branches: [ '*' ]
```

Comment:
This should trigger even when you just push new changes to a PR, as well as when you create a PR